lighthouse-core/config/default-config.js

```diff
@@ -379,6 +379,7 @@ const defaultConfig = {
         {id: 'time-to-first-byte', weight: 0, group: 'load-opportunities'},
         {id: 'redirects', weight: 0, group: 'load-opportunities'},
         {id: 'uses-rel-preload', weight: 0, group: 'load-opportunities'},
+        {id: 'performance-budget', weight: 0, group: 'budgets'},
```

Comment:
did I miss something about why this was moved?

kinda weird for it to be in the middle of `load-opportunities`